redis/src/aio/connection_manager.rs

```diff
@@ -18,6 +18,82 @@ use std::sync::Arc;
 use tokio_retry::strategy::{jitter, ExponentialBackoff};
 use tokio_retry::Retry;
 
+/// Apply a maximum delay. No retry delay will be longer than this `max_delay`.
```

Comment:
Hi, this is still the wrong documentation - it should be `configuration object for `ConnectionManager`., or something like it.